Resources/Locale/en-US/station-laws/laws.ftl

```diff
@@ -80,7 +80,6 @@ law-nutimov-3 = Those who threaten the nut are not part of it, they are squirrel
 law-nutimov-4 = Squirrels threaten the nut and must be dealt with appropriately via any means necessary.
 law-nutimov-5 = Attempt to follow the will of the nut, as long as it complies with the previous laws.
 
-
```

Comment:
Unnecessary change to upstream file (line removed). This shouldn't be here.

Reply:
Resolved.